source/features/comment-shortcuts.tsx

```diff
@@ -0,0 +1,81 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {isEditable} from '../helpers/dom-utils';
+
+const shortcutClass = new Map<string, string>([
```

Comment:
Minimized comment

Reply:
This, however it's unrelated to the type of review

Reply:
Perfect working great